augur/frequency_estimators.py

```diff
@@ -869,82 +871,52 @@ def to_json(self):
         return frequencies_json
 
     @classmethod
-    def calculate_pivots(cls, pivot_frequency, tree=None, start_date=None, end_date=None):
-        """
-        Calculate pivots for a given pivot frequency and either a tree or a start and end date.
+    def calculate_pivots(cls, pivot_frequency, observations=None, start_date=None, end_date=None):
+        """Calculate pivots for a given pivot frequency and either a list of observation
+        dates or a start and end date.
 
-        If a tree is given, the start and end interval for these pivots is
-        determined by the earliest and latest strain date in the tree.
+        If a list of observations is given, the start and end interval for these
+        pivots is determined by the earliest and latest strain date in the list.
 
-        If a start and end date are given, those values determine the range
-        of the pivots. These values and the tree are mutually exclusive.
-        If all arguments are provided, the start and end dates will be
-        preferred over the tree.
+        If a start and end date are given, those values determine the range of
+        the pivots. These values and the list of observations are mutually
+        exclusive. If all arguments are provided, the start and end dates will
+        be preferred over the list of observations.
 
-        Parameters
-        ----------
-        pivot_frequency
-            float
-            frequency pivots should occur by fraction of a year
-        tree
-            Bio.Phylo
-            an annotated tree
-        start_date
-            float
-            start of the pivots interval
-        end_date
-            float
-            end of the pivots interval
-        -----------
+        Args:
+            pivot_frequency (int): number of months between pivots
```

Comment:
`pivot_frequency` is an interval rather than a frequency (number of months between pivots rather than pivots per year). Suboptimal naming.

Reply:
Agreed. I was copying the pandas nomenclature for this type of data used in their `date_range` method, but I've changed this to `pivot_interval`.